priv/templates/extended_bin

```diff
@@ -129,25 +107,35 @@ if [ -z "$RELX_CONFIG_PATH" ]; then
     fi
 fi
 
+orig_vmargs_path="$VMARGS_PATH.orig"
+if [ $RELX_REPLACE_OS_VARS ]; then
+    # if there is no vm.args.orig then make a copy of the
+    # the original vm.args
+    if [ ! -f "$orig_vmargs_path" ]; then
+       mv "$VMARGS_PATH" "$orig_vmargs_path"
+    fi
+
+    # apply the environment variable substitution to vm.args.orig
+    # the result is saved to vm.args
+    awk '{while(match($0,"[$]{[^}]*}")) {var=substr($0,RSTART+2,RLENGTH -3);gsub("[$]{"var"}",ENVIRON[var])}}1' < "$orig_vmargs_path" > "$VMARGS_PATH"
+fi
+
```

Comment:
yeah, i had some doubts about this behaviour, i ended up going with run from modified vm.args, the reason being that the original probably has env variables that won't get replaced and most likely will cause the release start to fail, what are your thoughts?